### ISSUE TITLE
Rstudio-like indention in function calls

### ISSUE THREAD

**Issue by intiben** [r-lib/styler#616]
First of all I want to thank you for this very useful package. 

I have been working a lot with Rstudio and got used to the built-in indentation features, however I cannot find a way to reproduce it with styler. I would like to find a way to dynamically indent the content within a function to the open parenthesis and not to the `indent_by` argument.

This is what I would like to get (i.e. as in Rstudio)
```r
output <- mtcars %>%
   mutate(col1 = 1,
          col2 = 2)
```

This is what I am getting
```r
output <- mtcars %>%
   mutate(
     col1 = 1,
     col2 = 2
   )
```

I know that this is not necessarily the tidyverse style guide, but it is widely used and allows for a somehow compact coding style which can be convenient.

**Comment by lorenzwalthert** [comment]
Two things: 

1) As you said, such a rule does not comply with the tidyverse style guide.
2) There is an issue in which we wanted to achieve exactly that: https://github.com/r-lib/styler/issues/66 but I think we ended up solving it differently.
2) Beyond that, it's not trivial to implement it. I recall I tried that but to make it work for all cases was difficult and I stopped trying because of 1). If you want to fiddle yourself, the relevant code has only been commented out in https://github.com/r-lib/styler/commit/506e23aa5c662d765b80dd9004948053bc41d001.
3) Maybe we can consider adding an option to `tidyverse_style` after we figured out how to implement it properly in styler.

@hadley I think RStudio should not indent as described in this issue by default. This is IMHO by far the most common deviation from the tidyverse style guide I see in people's code, even after `=`. Is it worth opening up an issue in https://github.com/rstudio/rstudio?
